test.rb

```diff
@@ -0,0 +1,12 @@
+require 'opal'
+puts 'ready'
+
+t = String.new('bla')
```

Comment:
So, this is the only way we can get a mutable String, right? Perhaps we can also do `String#+`, maybe `String#dup` and maybe also by default for `dstr` (ie. expressions like `"test#{other}"`). If it can be pluggable, then we would just do `String.new` on it.

Reply:
Yes, currently its the only way. So everywhere we return a new_string it should be ::String.new(new_string)